projects/plugins/jetpack/extensions/blocks/button/button.php

```diff
@@ -211,6 +217,36 @@ function get_button_styles( $attributes ) {
 		$styles[] = 'max-width: 100%';
 	}
 
+	if ( $has_named_border_color ) {
+		$border_styles['color'] = "var:preset|color|{$attributes['borderColor']}";
```

Comment:
This doesn't seem to be working, although looking at the style engine output, `$attributes['borderColor']` only contains the last bit of the preset string.
Looking at what's happening over in core, that particular style doesn't seem to be output by the border support either; for preset colors the front end styling depends on the preset classname and its corresponding CSS rule.

I think we'll have to ensure the classname is output on the front end instead.

Reply:
Ok I've fixed it to output classname instead of styles.